clients/vscode-hlasmplugin/src/hlasmExternalFilesFtp.ts

```diff
@@ -106,9 +104,11 @@ export class HLASMExternalFilesFtp implements ExternalFilesClient {
             if (this.activeConnectionInfo)
                 return this.activeConnectionInfo;
 
-            const last = getLastRunConfig(this.context);
+            this.clientPool.closeClients();
+
+            const last = getLastRunConfig(this.context!);
```

Comment:
How do we lmpw that `this.context` is not null?

Reply:
Because it can be undefined only for testing purposes.

Reply:
These changes "leaked" from the next PR... I didn't see a reason to not keep them though.